### PR TITLE
Use extracted deploy-receivers

### PR DESCRIPTION
**Pull request by kraman**
Connect to strongloop-internal/scrum-nodeops#675

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,13 +48,14 @@
...
   "devDependencies": {
+    "concat-stream": "^1.4.7",
     "eslint": "^0.22.1",
     "jscs": "^1.11.3",
     "mktmpdir": "^0.1.1",
     "request": "^2.45.0",
     "semver": "^4.1.0",
     "shelljs": "^0.3.0",
-    "strong-build": "^2.0.0",
+    "strong-build": "^2.0.9",
```
Comment (sam-github) [inline package.json]: 2.0.9 doesn't exist, why did you change?

File: lib/drivers/docker/image.js [hunk 2]
```diff
@@ -1,13 +1,11 @@
 'use strict';
 
-var cicada = require('strong-fork-cicada');
 var Container = require('./container');
-var debug = require('debug');
+var DeployReceiver = require('strong-deploy-receivers');
```
Comment (sam-github) [inline lib/drivers/docker/image.js]: not added as dep in package, probably that's why all the ci is failling

also, check that strong-deploy-receivers is in ci, and in the scrum-nodeops board

**Comment by sam-github** [comment]
"extraced" -> "extracted"?

**Comment by sam-github** [comment]
Other than the comments on the metadata, LGTM

File: lib/drivers/docker/image.js (hunk 2 above)
Comment (rmg) [inline lib/drivers/docker/image.js]: strong-deploy-receivers also has to be added to CI and then merged so that it gets pushed to staging :-)

**Comment by kraman** [comment]
Waiting for strong-deploy-receivers modules to show up on staging

**Comment by sam-github** [comment]
Its not there, I pushed to master a minor change, hoping that will trigger it.

**Comment by rmg** [comment]
the jenkins job hadn't been deployed like I thought, but it has been fixed. strong-deploy-receivers@1.0.0 is now on staging.

**Comment by rmg** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

File: package.json [hunk 3]
```diff
@@ -78,6 +78,7 @@
     "rimraf": "^2.3.2",
     "serve-favicon": "^2.1.5",
     "strong-control-channel": "^2.0.0",
+    "strong-deploy-receivers'": "^1.0.0",
```
Comment (rmg) [inline package.json]: You've got a trailing `'`.. that's what is breaking CI, it's treating it as a path because `'` isn't a valid package name character.

**Comment by rmg** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please
